client/context/context.go

```diff
@@ -122,7 +122,8 @@ func createVerifier() tmlite.Verifier {
 		os.Exit(1)
 	}
 	node := rpcclient.NewHTTP(nodeURI, "/websocket")
-	verifier, err := tmliteProxy.NewVerifier(chainID, filepath.Join(home, ".gaialite"), node, log.NewNopLogger())
+	cacheSize := 10 // TODO What's a good value?
```

Comment:
What does the cache and it's size do here? Should we open an issue for this?

Reply:
I prefer no, I think we just keep the TODO.  I think we have too many issues, when a TODO in the code works just as well.  It's more efficient to have this convention.

Yes we can fix it at a later time, but I put TODOs in because they're not dangerous to ignore.  TODO means it doesn't warrant an issue/discussion (there *is* a cost to making an issue for everything) but it's worth noting.

Reply:
In other words, I don't think it's something worth worrying about now, and that we can revisit it later as we try to improve the system over time.

Reply:
Ok 👌